ui/components/app/confirm-page-container/confirm-page-container-navigation/confirm-page-container-navigation.component.js

```diff
@@ -72,7 +72,7 @@ const ConfirmPageContainerNavigation = () => {
           data-testid="previous-page"
           onClick={() => onNextTx(prevTxId)}
         >
-          <i className="fa fa-angle-left fa-2x" />
```

Comment:
You will have to import the `Icon` component and `ICON_NAMES` object and ensure the size matches the font awesome icons

```
import { Icon, ICON_NAMES } from '../../component-library';
```